src/components/30-organisms/footer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.2
+
+- Do not render an empty accordion on mobile if no content is given. #1836
```

Comment:
Would write: Only render accordion on mobile when content is given

Reply:
Wont fix. Minor.

Reply:
Why not? `ctrl C` + `ctrl V`
Double negative is difficult to read.